### ISSUE TITLE
targets/qtpy: incorrect volume name for UF2 mass storage device (MSD)

### ISSUE THREAD

**Issue by sago35** [tinygo-org/tinygo#1713]
I got the QTPy, but the volume name is different

I tried the following two bootloaders, but both had `QTPY_BOOT` instead of `QTPYBOOT`.

Is there a version where the Volume Name is  `QTPYBOOT` ?
In that case, we might want to extend the JSON to make multiple choices.

```
UF2 Bootloader v1.23.1-adafruit.1-328-gf06693a-dirty SFHWRO
Model: QT Py M0
Board-ID: SAMD21E18A-QTPy-v0
```

```
UF2 Bootloader v3.13.0 SFHWRO
Model: QT Py M0
Board-ID: SAMD21E18A-QTPy-v0
```

It seems to be `QTPY_BOOT` , at least in the latest bootloader.
https://github.com/adafruit/uf2-samdx1/blob/master/boards/QTPy_m0/board_config.h#L8

**Comment by deadprogram** [comment]
We should change to `QTPY_BOOT` I think that was error on my part.

**Comment by sago35** [comment]
OK.
I'll create a PR later.

**Comment by deadprogram** [comment]
This was released with v0.18 so now closing. Thank you!
